build.gradle

```diff
@@ -91,6 +91,10 @@ subprojects {
         implementation platform('com.fasterxml.jackson:jackson-bom:2.15.3')
         implementation platform('org.eclipse.jetty:jetty-bom:9.4.53.v20231009')
         implementation platform('io.micrometer:micrometer-bom:1.10.5')
+        implementation platform('software.amazon.awssdk:bom:2.17.17')
```

Comment:
You shouldn't need these four lines. 

We already get the BOM here:

https://github.com/opensearch-project/data-prepper/blob/d501af69c10b93efbb0c68abeeda4f5023323e18/build.gradle#L251

And for the following three dependencies, we don't need to choose which dependencies to pull it at the root project. Let the projects pull in what they need.

So you should be able to delete all four of these lines and make no changes to the root Gradle project.

Reply:
I had updated the changes as suggested